### ISSUE TITLE
Call an undefined method named "getAttribute" - Symfony & Akeneo PIM 5.0 & Sentry-bundle

### ISSUE THREAD

**Issue by xmxsm** [getsentry/sentry-symfony#537]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io) 

Which SDK and version?
PHP sentry-symfony 4.1.4, Akeneo 5.0.40, Symfony 4.4.25

### Steps to Reproduce

1. Install Akeneo: https://docs.akeneo.com/5.0/install_pim/docker/installation_docker.html
2. Install sentry-symfony bundle
3. When you going into http://localhost:8080/#/system_info - you will see 500 internal error.

### Expected Result

We should see system information page.

### Actual Result

We see 500 internal error page. In logs:

```
[2021-07-16 09:41:49] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "getAttribute" of class "Sentry\SentryBundle\Tracing\Doctrine\DBAL\TracingDriverConnection"." at /srv/pim/vendor/akeneo/pim-community-dev/src/Akeneo/Platform/Bundle/AnalyticsBundle/DataCollector/StorageDataCollector.php line 33 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Attempted to call an undefined method named \"getAttribute\" of class \"Sentry\\SentryBundle\\Tracing\\Doctrine\\DBAL\\TracingDriverConnection\". at /srv/pim/vendor/akeneo/pim-community-dev/src/Akeneo/Platform/Bundle/AnalyticsBundle/DataCollector/StorageDataCollector.php:33)"} []
[2021-07-16 09:41:49] request.CRITICAL: Exception thrown when handling an exception (InvalidArgumentException: Unable to find template "" (looked into: /srv/pim/vendor/symfony/twig-bridge/Resources/views/Form). at /srv/pim/vendor/symfony/twig-bridge/TwigEngine.php line 137) {"exception":"[object] (InvalidArgumentException(code: 0): Unable to find template \"\" (looked into: /srv/pim/vendor/symfony/twig-bridge/Resources/views/Form). at /srv/pim/vendor/symfony/twig-bridge/TwigEngine.php:137, Twig\\Error\\LoaderError(code: 0): Unable to find template \"\" (looked into: /srv/pim/vendor/symfony/twig-bridge/Resources/views/Form). at /srv/pim/vendor/twig/twig/src/Loader/FilesystemLoader.php:265)"} []
```

After disable bundle - we will see system information page correctly.

**Comment by Jean85** [comment]
This seems something cause by the DBAL tracing feature. You can apply this workaround in the config:
```
sentry:
  tracing:
    dbal:
      enabled: false
```

Also, this could be an Akeneo bug, because `getAttribute` doesn't seem to be a DBAL connection method...

**Comment by Jean85** [comment]
Yes, I can confirm: https://github.com/akeneo/pim-community-dev/blob/2e2f18679ce33f1c050d5fbde4e7e700c5ece300/src/Akeneo/Platform/Bundle/AnalyticsBundle/DataCollector/StorageDataCollector.php#L33

This could be fixed (partially) by #536, but in general, with DBAL 3.0, you could have multiple wrappings, so it's not guaranteed to work.

**Comment by xmxsm** [comment]
Thank you for information. The solution from [here](https://github.com/getsentry/sentry-symfony/issues/537#issuecomment-881381415) works for me.
